mock/py/mockbuild/buildroot.py

```diff
@@ -103,7 +103,13 @@ def __init__(self, config, uid_manager, state, plugins, bootstrap_buildroot=None
         self.build_log = getLog("mockbuild.Root.build")
         self.logging_initialized = False
         self.chroot_was_initialized = False
-        self.preexisting_deps = []
+
+        build_requires = config["build_requires"] or []
+        if is_bootstrap:
+            self.preexisting_deps = []
+        else:
+            self.preexisting_deps = build_requires
```

Comment:
Hmm, why?
If you wipe away this whole if-section and set config_opts['bootstrap_command_line_addition_packages'] in config.py to empty list then you get the freedom to admin to override this value in config. Which is impossible with this hardcoded style.

Reply:
I can't imagine a reason to put this into configuration files... therefore I intentionally made this so it can not be configured at all.

In all situations where I got one would use 'chroot_additional_packages' instead.